### ISSUE TITLE
Template: Link missing a text alternative

### ISSUE THREAD

**Issue by syarwick** [loyolauniversitychicago/loyola-template-4.0#94]
In the footer, if the email field is not filled out, a blank link populates but does not have any alternative text. 

[From SiteImprove](https://my2.siteimprove.com/Inspector/59595/A11Y/Page?pageId=77217112710&impmd=7C080833A21920C8A763A2CEAF1D26A7&issue=cantTell+failed&wcag=two&decision=dismissed+falsePositive+none#/sia-r11/failed/c3o0RHFUaHM=;ODA3YWMwMDQ=)

**Comment by syarwick23** [comment]
Fixed in T4 content type
